matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/send/SendService.kt

```diff
@@ -29,20 +29,22 @@ interface SendService {
 
     /**
      * Method to send a text message asynchronously.
+     * The text to send can be a Spannable and contains special spans (UserMentionSpan) that will be translated
+     * by the sdk into pills.
      * @param text the text message to send
      * @param msgType the message type: MessageType.MSGTYPE_TEXT (default) or MessageType.MSGTYPE_EMOTE
      * @param autoMarkdown If true, the SDK will generate a formatted HTML message from the body text if markdown syntax is present
      * @return a [Cancelable]
      */
-    fun sendTextMessage(text: String, msgType: String = MessageType.MSGTYPE_TEXT, autoMarkdown: Boolean = false): Cancelable
+    fun sendTextMessage(text: CharSequence, msgType: String = MessageType.MSGTYPE_TEXT, autoMarkdown: Boolean = false): Cancelable
 
     /**
      * Method to send a text message with a formatted body.
      * @param text the text message to send
      * @param formattedText The formatted body using MessageType#FORMAT_MATRIX_HTML
      * @return a [Cancelable]
      */
-    fun sendFormattedTextMessage(text: String, formattedText: String): Cancelable
+    fun sendFormattedTextMessage(text: String, formattedText: String, msgType: String = MessageType.MSGTYPE_TEXT): Cancelable
```

Comment:
update javadoc for msgType (copy paste from previous method)

Reply:
Done